Makefile

```diff
@@ -1,9 +1,58 @@
 TOOL_VERSION = 2.1.12
 TEST_DATA = test/simpleTestForGenerator
-
+GIT_USERNAME=License Publisher (maintained by Gary O'Neall)
+GIT_EMAIL=gary@sourceauditor.com
+LICENSE_DATA_REPO=https://github.com/goneall/license-list-data.git
+LICENSE_DATA_REPO_NO_SCHEME=github.com/goneall/license-list-data.git
+LICENSE_DATA_URL=https://$(GITHUB_TOKEN)@$(LICENSE_DATA_REPO_NO_SCHEME)
+LICENSE_OUTPUT_DIR=.tmp
+VERSION=$(shell git describe --always || echo 'UNKNOWN')
+RELEASE_DATE=$(shell date -I)
+COMMIT_MSG="Automatic update of license list data from license XML repository version $(VERSION)"
```

Comment:
The output repo shouldn't care whether the source repo uses XML or not.  And we don't need to say what's being updated, because it will always be the bulk of the output repo.  I recommend a commit message closer to:

```
COMMIT_MSG = Build $(VERSION) using publisher $(TOOL_VERSION)
```